packages/react/src/components/Forms/Input/index.scss

```diff
@@ -1,23 +1,116 @@
-.c__input {
-  height: 2rem;
-  padding: 0 14px;
-  box-sizing: border-box;
-  border-radius: 2px;
-  border: 1px var(--c--theme--colors--greyscale-300) solid;
+.c__input__wrapper {
+
+  @mixin text-style {
+    font-size: var(--c--components--forms-input--font-size);
+    font-weight: var(--c--components--forms-input--font-weight);
+  }
+
+  border-radius: var(--c--components--forms-input--border-radius);
+  border-width: var(--c--components--forms-input--border-width);
+  border-color: var(--c--components--forms-input--border-color);
+  border-style: var(--c--components--forms-input--border-style);
+  display: flex;
+  align-items: center;
   transition: border var(--c--theme--transitions--duration) var(--c--theme--transitions--ease-out);
-  color: var(--c--theme--colors--greyscale-800);
-  font-size: var(--c--theme--font--sizes--m);
+  padding: 0 1rem;
+  gap: 1rem;
+  cursor: text;
+  color: var(--c--components--forms-input--color);
+  box-sizing: border-box;
+  height: 3.5rem;
```

Comment:
If we don't set it here, where?

Reply:
It could be lead by the padding and font size.

Reply:
What I like with setting height that way is that we only rely on flex properties in order to center the content. Plus the fact that it's easier to exactly match the sketches height which I find less error-prone and clearer, it also tells other developers that we want exactly this height. 

Also, it perfectly aligns with other components of the DS without having to make adjustments on each. 

Otherwise, using padding often results in gropingly finding this kind of magic values padding: `0.825rem 1rem`.